etc/config-template.toml

```diff
@@ -12,6 +12,8 @@ addr = "127.0.0.1:20160"
 #advertise-addr = ""
 # set the path to rocksdb directory.
 data-dir = "/tmp/tikv/store"
+# set the path to raft rocksdb directory.
+raft-data-dir = "/tmp2/tikv/store"
```

Comment:
If we don't set raft-data-dir, can we use `/tmp/tikv/store/raft` directly for the raft?